dev/docker/ranger/start-ranger-services.sh

```diff
@@ -15,12 +15,6 @@ sed -i "s/rangerAdmin_password=/rangerAdmin_password=${RANGER_PASSWORD}/g" /opt/
 sed -i "s/rangerTagsync_password=/rangerTagsync_password=${RANGER_PASSWORD}/g" /opt/ranger-admin/install.properties
 sed -i "s/rangerUsersync_password=/rangerUsersync_password=${RANGER_PASSWORD}/g" /opt/ranger-admin/install.properties
 sed -i "s/keyadmin_password=/keyadmin_password=${RANGER_PASSWORD}/g" /opt/ranger-admin/install.properties
-sed -i 's/check_java_version/#check_java_version/g' /opt/ranger-admin/setup.sh
-sed -i 's/#check_java_version()/check_java_version()/g' /opt/ranger-admin/setup.sh
-sed -i 's/check_db_connector/#check_db_connector/g' /opt/ranger-admin/setup.sh
-sed -i 's/#check_db_connector()/check_db_connector()/g' /opt/ranger-admin/setup.sh
-sed -i 's/copy_db_connector/#copy_db_connector/g' /opt/ranger-admin/setup.sh
-sed -i 's/#copy_db_connector()/copy_db_connector()/g' /opt/ranger-admin/setup.sh
```

Comment:
Cool!
Currently, we don't need to manualy disable these functions in the `setup.sh`?
Why? Because we download JDBC jar in to correct path?

Reply:
Because in `setup.sh`, it will use `check_db_connector()` to check connector jar at `CONNECTOR_JAR`,
and use `copy_db_connector()` to copy it to the correct location.

The default value of `CONNECTOR_JAR` is `/usr/share/java/mysql-connector-java.jar`

https://github.com/datastrato/apache-ranger/blob/release-ranger-2.4.0/security-admin/scripts/setup.sh#L55

https://github.com/datastrato/apache-ranger/blob/50ad9c19e62f5aa6b2888b17437aa6fb1e2c0a36/security-admin/scripts/install.properties#L38